api/src/modules/scenario-interventions/dto/create.scenario-intervention.dto.ts

```diff
@@ -54,9 +56,74 @@ export class CreateScenarioInterventionDto {
   @ApiPropertyOptional()
   percentage?: number;
 
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  materialsIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  businessUnitsIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  suppliersIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  adminRegionsIds: string[];
+
+  @IsUUID(4, { each: true })
+  @IsOptional()
+  @ApiPropertyOptional()
+  sourcingLocationsIds: string[];
+
   @IsString()
   @IsOptional()
   @IsJSON()
   @ApiPropertyOptional()
   newIndicatorCoefficients?: JSON;
+
+  @IsUUID()
```

Comment:
If this is a expected DTO to be received as payload from the consumer, I think this isn't right as no consumer can provide a UUID as these are generated once something is persistend in the backend (DB)